src/Event/Newsletter/Mailchimp/TemplateExportResolveProviderHandlerEvent.php

```diff
@@ -49,7 +49,7 @@ public function getName()
     /**
      * @return Mailchimp|null
      */
-    public function getProviderHandler(): Mailchimp
+    public function getProviderHandler()
```

Comment:
why not use?

```suggestion
    public function getProviderHandler(): ?Mailchimp
```

Reply:
since pimcore 5.4 already requires php 7.1 definetly possible